src/library/f3d_window_impl_standard.cxx

```diff
@@ -98,6 +101,12 @@ window_impl_standard::window_impl_standard(const options& options, bool offscree
 {
   this->Internals->RenWin->SetMultiSamples(0); // Disable hardware antialiasing
   this->Internals->RenWin->SetOffScreenRendering(offscreen);
+
+  this->Internals->RenWin->Start();
```

Comment:
This is indeed needed. Before your change this was always handled by the interactor before we set the window name. In any case, this is better.

Reply:
ok